app/src/main-process/menu/build-default-menu.ts

```diff
@@ -177,8 +177,8 @@ export function buildDefaultMenu(
         // Ctrl+Alt is interpreted as AltGr on international keyboards and this
         // can clash with other shortcuts. We should always use Ctrl+Shift for
         // chorded shortcuts, but this menu item is not a user-facing feature
-        // so we are going to keep this one around and save Ctrl+Shift+R for
-        // a different shortcut in the future...
+        // so we are going to keep this one around. Ctrl+Shift+R shortcut is
+        // used for renaming a branch which is a user facing feature.
```

Comment:
This comment confused me because I thought it was related to the change you made. I think this line is fine to be removed.

Reply:
👍